imgix/react-imgix
# test: add integration tests with enzyme, and use jest instead of mocha

## Description

Previously this project used `skin-deep` to test this component using shallow rendering. Although this was useful for a lot of testing scenarios, it doesn't allow integration tests (i.e. interacting with an actual DOM) to be created. Using a testing library which allows DOM integration tests permits a more comprehensive test suite to be created. Moving to using enzyme allows this.

### New Feature

- [x] The PR title is in the conventional commit format: e.g. `feat(<area>): added new way to do this cool thing #issue-number`

## Steps to Test

Steps:

1.  Clone this PR and run `npm install`
2.  Run `npm run test` and observe that all the tests now use Jest
